<div id="instructions" class="panel panel-default">
    <div class="panel-body">
        <div class="page-header">
            <h1>CoReBIN <small>Comparison and Evaluation of metagenomic binning</small></h1>
        </div>

        <ul class="nav nav-pills nav-stacked pull-right" id="home-navbar">
            <li role="presentation"><a href="#info">Info</a></li>
            <li role="presentation"><a href="#instructions">Instructions</a></li>
        </ul>

        <div data-spy="scroll" data-target="#home-navbar" style="position: relative">
            <h4 id="info">Info</h4>
            <p>
                CoReBIN is a user-friendly web-based tool to compare the results of different binning methods and to aid manual refinement of the bins. The overall binning result of two different methods can be visually compared in a chord diagram. Individual bins can be inspected using GC%, coverage and tetranucleotide frequencies. Duis posuere, dolor quis tempor porttitor, augue sapien fermentum diam, sed sollicitudin augue neque at tortor.              
            </p>
            
        	<h4 id="data">Input data</h4>
        	<p>
                <ul>
                <li>Metagenomic assembly (fasta format)</li>
                <li>Coverage (see format details below) <i>not required</i>
                <li>One or more bin sets (see format details below)</li>
                </ul>
            </p>

            <h4 id="instructions">Usage</h4>
            <p>First, upload your metagenomic assembly. It is advised to also upload a table with coverage information on your assembly. Next you can upload the result of your binning. To do so, you need a file in which the contig ID if followed by the bin number.
            </p>
            
         	<h4 id="data">Metagenomic binning programs</h4>
        	<p>
        	Here are some suggestions of recently published metagenomic binning programs and methods:
                <ul>
                <li><a href="https://github.com/BinPro/CONCOCT">CONCOCT</a>: Alneberg, Johannes, et al. "Binning metagenomic contigs by coverage and composition." <i>Nature methods</i> 11.11 (2014): 1144-1146.</li>
                <li><a href="https://bitbucket.org/berkeleylab/metabat">MetaBAT</a>: Kang, Dongwan D., et al. "MetaBAT, an efficient tool for accurately reconstructing single genomes from complex microbial communities." <i>PeerJ</i> 3 (2015): e1165.</i>
                <li><a href="http://ecogenomics.github.io/GroopM/">GroopM</a>: Imelfort, Michael, et al. "GroopM: an automated tool for the recovery of population genomes from related metagenomes." <i>PeerJ</i> 2 (2014): e603.</li>
				<li><a href="https://sourceforge.net/projects/maxbin/">MaxBin 2.0</a>: Wu, Yu-Wei, Blake A. Simmons, and Steven W. Singer. "MaxBin 2.0: an automated binning algorithm to recover genomes from multiple metagenomic datasets." <i>Bioinformatics</i> (2015): btv638.</li>
				<li><a href="http://claczny.github.io/VizBin/">VizBin</a>: Laczny, Cedric C., et al. "VizBin-an application for reference-independent visualization and human-augmented binning of metagenomic data." <i>Microbiome</i> 3.1 (2015): 1.</li>
                </ul>
            </p>           
            
        </div>
    </div>
</div>